### PR TITLE
DPC-1252: Create new implementer portal

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-1252](https://jira.cms.gov/browse/DPC-1252)

Create a new directory in the repo for the admin portal (/dpc-impl)

### Proposed Changes

- Start new Rails app scaffolding in the new directory. Use admin app as a reference.
- Make sure Rails 6 uses NPM (not yarn, the default package manager)
- Set up basic layout (nav bar, etc.)

### Change Details

- Added `dotenv` gem
- Set up portal landing page
- Set up javascripts link
- Set up styling with `sass-rails`

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

- Create a new directory in the repo for the admin portal (/dpc-impl)
- Start a new Rails app scaffolding in the new directory. Use the new admin app as a reference.
  - Make sure Rails 6 uses NPM (not yarn, which is the default)
- Set up basic layout (nav bar, etc)

Screen Shot 2021-04-08 at 2 38 22 PM

**Responsive**
Screen Shot 2021-04-08 at 2 38 32 PM

Screen Shot 2021-04-08 at 2 38 36 PM

### CODE REVIEW

File: dpc-impl/.gitignore [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Ignore uploaded files in development.
+/storage/*
+!/storage/.keep
+
+/public/assets
+.byebug_history
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+/public/ig/**/*
```
Comment (dhgreene) [inline dpc-impl/.gitignore:25]: This isn't relevant any more

File: dpc-impl/Gemfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+gem 'rails', '~> 6.1.3'
+gem 'dotenv-rails', groups: [:development, :test]
+gem 'pg', '~> 1.1'
+gem 'puma', '~> 5.0'
+gem 'sass-rails', '>= 6'
+gem 'webpacker', '~> 5.0'
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.7'
+# Use Redis adapter to run Action Cable in production
+# gem 'redis', '~> 4.0'
```
Comment (dhgreene) [inline dpc-impl/Gemfile:15]: Should we just delete these commented-out gems?

Comment (Sun-Mountain) [reply dpc-impl/Gemfile:15]: Will do! (And also will delete the ig path)

**Review by dhgreene — APPROVED** [review]
A clean slate 😄

**Pull request merged**